react/src/data/accessor.ts

```diff
@@ -12,6 +12,16 @@ export class ModelsAccessor<M, F> {
                 modelsWrapper.hookModel(modelName)
             ]
         }))
+
+        // expand wildcard
+        this.dependencies = this.dependencies.flatMap(d => {
+            if(d.fieldName === '*') {
```

Comment:
Magic syntax like this is hard to find. What about having a keyword, e.g. _ALL_. This will make explicitly coupling a bit easier and make the app less rigid.

Reply:
So you want `model.ALL` vs `model.*`?

Reply:
My only thought after a second is that technically we could have a field name collide with our magic syntax here. Any field named `all` would fail and to prevent a very confusing error we would need a check for this in an unrelated part of the app.

Reply:
My bad I edited my comment `_ALL_` or some variant that is searchable that would not collide with field names. This would be pretty easy, since we already restrict field name syntax. In the future, someone will want to search for the concept of `*`, which is very hard to find. Therefore, it's better to have a longer string that is globally unique and perhaps self documenting (even better `_ALL_FIELDS_`) for cases like this.